swig/cpp/src/Session.hpp

```diff
@@ -162,27 +163,27 @@ class Callback
     virtual ~Callback();
 
     /** Wrapper for [sr_module_change_cb](@ref sr_module_change_cb) callback.*/
-    virtual int module_change(S_Session session, const char *module_name, sr_notif_event_t event, void *private_ctx) {return SR_ERR_OK;};
+    virtual int module_change(S_Session /* session */, const char* /* module_name */, sr_notif_event_t /* event */, void* /* private_ctx */) {return SR_ERR_OK;};
```

Comment:
This (and any other actual definitions of the method bodies) are the place to modify.

The "proper" fix is the `[[maybe_unused]]` attribute, but that's only become available in C++17 which we cannot reasonably require.

If you insist on fixing this, please only change these actual method bodies.

Reply:
You are right. It is done. I insist because it is cluttering up my build output and prevents me from seeing my own warnings.

Reply:
Please squash these commits together and force-push so that the git history stays tidy.

Also, your workflow might benefit from the compiler's `-isystem` so that it treats sysrepo's headers as a part of the system and not subject to your project's policies about warnings, `-Werror`, etc.